mono/mini/interp/interp.c

```diff
@@ -2111,8 +2111,10 @@ ves_exec_method_with_context (MonoInvocation *frame, ThreadContext *context, uns
 	unsigned char *vt_sp;
 	unsigned char *locals;
 	MonoError error;
-	MonoObject *o = NULL;
+	MonoObject *o = NULL, *this_arg;
 	MonoClass *c;
```

Comment:
Don't think this will save much stack space, gcc/clang is smart enough to share stack slots if their live range doesn't overlap.